regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -238,7 +241,24 @@ var MainView = Backbone.View.extend({
 
         // change focus to main content area when new sections are loaded
         $('.section-focus').focus();
-        $('table').stickyTableHeaders();
+        this.setHandlers();
+    },
+
+    initHandlers: function() {
+        $.extend($.fn.dataTable.defaults, {
+                paging: false,
+                searching: false,
+                scrollY: 400,
+                scrollX: true,
+                info: false
+        });
+    },
+
+    setHandlers: function() {
+        $('table', this.$el).each(function(index, element) {
+            $(element).DataTable();
+        });
```

Comment:
The rest of the app uses `this.$el.find(selector)` rather than `$(selector, this.$el)`. Is there a reason to break the mold here?

Unrelated, but can we skip the each? I.e. would `$(selector, this.$el).DataTable()` work?